tests/integration/webclient/pom.xml

```diff
@@ -93,4 +93,49 @@
         </dependency>
     </dependencies>
 
+    <build>
```

Comment:
I think, it should be safe to remove this. I have tested to run the tests without it and it runs as expected. Could you please double-check?

Reply:
It depends in the order in which tests are executed, which can vary. Was ConnectionClose executed at the end of your run? Otherwise, it can shut down the server midstream.